data/strings/types_strings.txt

```diff
@@ -21119,7 +20864,8 @@
     zh-Hant = 百貨公司
 
   [type.shop.doityourself]
-    en = Hardware Store
+    en = DIY Store
```

Comment:
Nobody calls these places DIY stores in the US.    Basically there is "hardware store" where you can buy not only nuts and bolts, but all sorts of plumbing and electrical parts, some car parts, limited building supplies, paint, pest control, power tools, gas grills, generators.  These are not giant.  Then there are "home improvement warehouse stores", which is pronounced "Home Depot" or "Lowe's".  These are enormous, and are basically a giant hardware store, a lumberyard, and a garden center all in one.

So I think it's "Home improvement store".

Reply:
yeah that sounds right, updated